tests/should_succeed/compiler_cli.jou

```diff
@@ -88,7 +88,7 @@ def main() -> int:
 
     # Compile a GUI program
     if not (WINDOWS or MACOS):
-        ret = system("./jou -o /dev/null --linker-flags \"-lX11\" examples/x11_window.jou")
+        ret = system("./jou -o /dev/null --linker-flags \"-L/usr/X11R7/lib -lX11\" examples/x11_window.jou")
```

Comment:
Would be nice, if netbsd specific stuff was under `if NETBSD`. That way it's clear why the weird folder path. IMO it doesn't matter if you copy/paste rest of the command twice.

Reply:
It’s not weird, more like the Unix standard :)